modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/service/impl/FriendlyURLEntryLocalServiceImpl.java

```diff
@@ -515,6 +594,12 @@ public FriendlyURLEntry updateFriendlyURLEntry(
 		_updateFriendlyURLEntryLocalizations(
 			friendlyURLEntry, classNameId, urlTitleMap);
 
+		// Asset
+
+		if (serviceContext != null) {
+			_updateAsset(friendlyURLEntry, serviceContext);
+		}
```

Comment:
Why not add the null check inside `_updateAsset`?

Reply:
Done :)